src/maheshKJun20/StringExamples/SumOfDigitsInString.java

```diff
@@ -0,0 +1,20 @@
+package maheshKJun20.StringExamples;
+
+public class SumOfDigitsInString {
+	
+	void displaySumOfDigits(String str) {
+		int sum = 0;
+		for (int index=0;index<str.length();index++) {
+			if (Character.isDigit(str.charAt(index))){
+				sum += Integer.parseInt(String.valueOf(str.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));